local/bin/py/build/update_pre_build.py

```diff
@@ -150,6 +151,7 @@ def build_documentation(self):
 def main(disable_cache_on_retry=False):
     # Those hard-written variables should be set in the Makefile config later down the road.
     build_configuration_file_path = getenv("CONFIGURATION_FILE")
+    pull_config_allowlist_file_path = "./local/bin/py/build/configurations/pull_config_allowlist.yaml"
```

Comment:
This pathing will change once it's in the ci repo since it will not be kept in the docs repo or on locals